### PR TITLE
Update documentation

### LINKED ISSUE #20: Link to install CompatHelper
- https://github.com/bcbi/CompatHelper.jl

### LINKED ISSUE #22: Add commands to contribute from fork
```bash
[mforets@localhost dev]$ cd Polyhedra
[mforets@localhost Polyhedra]$ git remote -v
origin  https://github.com/JuliaPolyhedra/Polyhedra.jl.git (fetch)
origin  https://github.com/JuliaPolyhedra/Polyhedra.jl.git (push)
[mforets@localhost Polyhedra]$ git remote add upstream https://github.com/JuliaPolyhedra/Polyhedra.jl.git
[mforets@localhost Polyhedra]$ git remote set-url origin https://github.com/mforets/Polyhedra.jl.git
[mforets@localhost Polyhedra]$ git remote -v
origin  https://github.com/mforets/Polyhedra.jl.git (fetch)
origin  https://github.com/mforets/Polyhedra.jl.git (push)

upstream        https://github.com/JuliaPolyhedra/Polyhedra.jl.git (fetch)
upstream        https://github.com/JuliaPolyhedra/Polyhedra.jl.git (push)
```

### LINKED ISSUE #29: ColPrac
Consider adding links to https://github.com/SciML/ColPracd 

discourse thread https://discourse.julialang.org/t/colprac-contributors-guide-on-collaborative-practices-for-community-packages/41850

### LINKED ISSUE #30: Add Atom tips
- language-julia
- language-markdown
- latex-completions
- markdown-preview-plus
- platformio-ide-terminal
- teletype

### PR DESCRIPTION
**Pull request by mforets**
Closes #29.
Closes #22.
Closes #30.
Closes #20.

### CODE REVIEW

**Comment by mforets** [comment]
This is ready from my side -- I plan to address the other open issues in another PR.

File: docs/src/release.md [hunk 1]
```diff
@@ -67,3 +69,38 @@ create a release on Github with the corresponding release notes.
...
+
+## Registrator bot
+
+The [Registrator.jl](https://github.com/JuliaRegistries/Registrator.jl) bot is a
+github app that automates creation of registration pull requests for your Julia packages
+to the General registry. To install, just click on the installation button that can be
+found on `Registrator.jl`'s documentation. Once installed, the usual procedure to
+register a new release is:
+
+First, install the app on your package(s) as mentioned above. The procedure for registering a new package is the same as for releasing a new version.
```
Comment (schillic) [inline docs/src/release.md]:
```suggestion
The procedure for registering a new package is the same as for releasing a new version.
```
This sentence was already covered by the paragraph before.

File: docs/src/workflow.md [hunk 2]
```diff
@@ -79,12 +80,56 @@ $ julia --color=yes docs/make.jl
...
+of X, and then creates a PR to be merged by the maintainer of package X.
+You have to properly setup the branches to do so:
+
+- `origin`   -- is usually set to the fork of the project
+- `upstream` -- is usually set to the main project
+
+For example, suppose that user `mforets` would like to setup the pacakge `Polyhedra`
+working from a fork. The git command `git remote -v` prints all the remote repositories,
+and you can modify them with `git remote add` to add a new one and
+`git remote set-url` to change the URL of an existing repository (it also possible to
```
Comment (schillic) [inline docs/src/workflow.md]:
```suggestion
`git remote set-url` to change the URL of an existing repository (it is also possible to
```

File: docs/src/workflow.md [hunk 3]
```diff
@@ -79,12 +80,56 @@ $ julia --color=yes docs/make.jl
...
+origin  https://github.com/mforets/Polyhedra.jl.git (fetch)
+origin  https://github.com/mforets/Polyhedra.jl.git (push)
+
+upstream        https://github.com/JuliaPolyhedra/Polyhedra.jl.git (fetch)
+upstream        https://github.com/JuliaPolyhedra/Polyhedra.jl.git (push)
+```
+
+## Editors
+
+There is no official IDE for Julia. For those using Atom editor, there are several
```
Comment (schillic) [inline docs/src/workflow.md]:
```suggestion
There is no official IDE for Julia. For those using the [Atom](https://atom.io/) editor, there are several
```

**Review by schillic — APPROVED** [review]
Thanks!

**Pull request merged**